src/SFA.DAS.Forecasting.Api.UnitTests/Models/WhenCastingApprenticeshipCourseFromMediatorType.cs

```diff
@@ -23,11 +23,13 @@ public class WhenCastingApprenticeshipCourseFromMediatorType
                     .Excluding(s => s.TypicalDuration)
                     .Excluding(s => s.StandardDates)
                     .Excluding(s => s.IsActive)
+                    .Excluding(s => s.StandardUId)
+                    .Excluding(s => s.LarsCode)
                 );
 
             response.Duration.Should().Be(source.TypicalDuration);
             response.FundingCap.Should().Be(source.MaxFunding);
-            
+            response.Id.Should().Be(source.LarsCode.ToString());
```

Comment:
Was this wrongly returning a string type on Id?

Reply:
No I had a look at this wondering if it was a mistake. The `ApprenticeshipCourse` object can be populated by either a Framework or a Standard, The ID of a Framework is a string, so due to the re-use of an object, both the StandardId and FrameworkId fit into the string Id field so I followed suit.